data/sources/21_pois.yaml

```diff
@@ -0,0 +1,28 @@
+parabelrutsche:
+  parent: "mi"
+  name: "Parabelrutsche"
+  coords: { lat: 48.26254, lon: 11.66796 }
+
+  usage:
+    name: "Gravitatives Transportmittel"
+
+  props:
+    generic:
+      - name:
+          de: "Formel"
+          en: "Formula"
+        text: "z = y = hx²/d²"
+
+# === Validierungsautomaten ===
+validierungsautomat-1:
```

Comment:
Could we add all the different machines:
The [documentation for which building they are](https://www.tum.de/studium/bewerbung/infoportal-einschreibung/studentcard) in is clear enough for most, exact coordinates could then be added via the normal feedback..

Reply:
I have added all machines and their coordinates if I could find them out (if not there's sometimes a coment; for the MI thanks to [the Campus Tour](https://youtu.be/rQNNL1tvmDQ?t=10) ^^ ). For two machines I couldn't find out their number from the [HTML source of the status page](https://campus.tum.de/valistatus/) like for the others:
- TUM Campus Straubing Bibliothek
- TUM School of Education

Reply:
The CS is here: https://nav.tum.de/building/2927
The "Edu" one is here: https://nav.tum.de/room/2907.01.130

Source: https://www.tum.de/studium/bewerbung/infoportal-einschreibung/studentcard
(Section Standorte der Validierungsstationen)

Reply:
Oh sorry I was not clear maybe. I couldn't find out their number like "Station Nr. 42", so their IDs are currently `validierungsautomat-straubing` and not `validierungsautomat-42`